Backoffice/RestoreEntity/Strategies/RestoreContentStrategy.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace OpenOrchestra\Backoffice\RestoreEntity\Strategies;
+
+use OpenOrchestra\Backoffice\RestoreEntity\RestoreEntityInterface;
+use OpenOrchestra\BackofficeBundle\Manager\ContentManager;
+use OpenOrchestra\ModelInterface\Model\ContentInterface;
+
+/**
+ * Class RestoreContentStrategy
+ */
+class RestoreContentStrategy implements RestoreEntityInterface
+{
+    /**
+     * @var ContentManager
+     */
+    protected $nodeManager;
+
+    /**
+     * @param ContentManager $contentManager
+     */
+    public function __construct(ContentManager $contentManager)
+    {
+        $this->contentManager = $contentManager;
+    }
+
+    /**
+     * @param mixed $entity
+     *
+     * @return bool
+     */
+    public function support($entity)
+    {
+        return $entity instanceof ContentInterface;
+    }
+
+    /**
+     * @param ContentInterface $content
+     */
+    public function restore($content)
+    {
+        $this->contentManager->restoreContent($content);
```

Comment:
Yes but where do you put them, I can put in ContentStrategy or NodeStrategy but in this case, they will use components of Symfony so they will not be in the business logic layer